openpype/hosts/maya/plugins/load/load_as_template.py

```diff
@@ -0,0 +1,40 @@
+from openpype.lib import (
+    BoolDef
+)
+from openpype.pipeline import (
+    load,
+    registered_host
+)
+from openpype.hosts.maya.api.workfile_template_builder import (
+    MayaTemplateBuilder
+)
+
+
+class LoadAsTemplate(load.LoaderPlugin):
+    """Load workfile as a template """
+
+    families = ["workfile"]
+    label = "Load as template"
+    representations = ["ma", "mb"]
+    icon = "wrench"
+    color = "#775555"
+    order = 10
+
+    options = [
+        BoolDef("keep_placeholders",
+                label="Keep Placeholders",
+                default=False),
+        BoolDef("create_first_version",
+                label="Create First Version",
+                default=False),
+    ]
+
+    def load(self, context, name, namespace, data):
+
```

Comment:
Out of curiosity, why you have first lines empty?

Reply:
Personal preference I suppose when I am reading the code. Want me to remove it?